src/general/OptionsMenu.cs

```diff
@@ -222,17 +274,44 @@ public void SetSettingsFrom(Settings settings)
 
     private void SetSettingsTab(string tab)
     {
+        // Convert from the string binding to an enum.
+        SelectedOptionsTab newSelection;
+        switch (tab)
+        {
+            case "graphics":
+                newSelection = SelectedOptionsTab.Graphics;
```

Comment:
I think C# has an inbuilt way to convert string to an enum entry, might need to rename the things in the GUI scene to match the names exactly.